website/docs/cdktf/release/index.mdx

```diff
@@ -0,0 +1,36 @@
+---
+page_title: Releases
+description: >-
+  We ship a CDK for Terraform release each month. Find a list of recent releases and a link to our changelog.
+---
+
+# Releases
+
+-> **Note:** CDK for Terraform is currently in [beta](#project-maturity-and-production-readiness).
+
+We ship a new Terraform Enterprise release each month. Recent releases are listed below. Our [CHANGELOG on github](https://github.com/hashicorp/terraform-cdk/blob/main/CHANGELOG.md) contains details about bug fixes, new features, and deprecations.
+
+- [release name](link to changelog)
+
+## Upgrade Guides
+
+- [Upgrading to v0.7](/cdktf/release/upgrade-guide-v0-7)
```

Comment:
So we're going to move these from the GitHub repo to standalone docs pages?

Reply:
yes! I will add a line about this - great idea!